### PR TITLE
adjust hypershift-mce-power-conformance workflow

### PR DESCRIPTION
**Pull request by LiangquanLi930**
Adjust the `hypershift-mce-power-conformance` workflow
Change the way of obtaining osImages to support more versions

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-mce-power-conformance

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-mce-power-conformance

File: ci-operator/step-registry/hypershift/mce/power/conformance/hypershift-mce-power-conformance-workflow.yaml [hunk 1]
```diff
@@ -8,8 +8,10 @@ workflow:
       - ref: ingress-aws-nlb-manifest
       - chain: ipi-aws-pre
       - ref: hypershift-mce-install
+      - ref: hypershift-mce-conf-os-images
       - ref: hypershift-mce-power-create
     test:
+      - ref: hypershift-mce-agent-info
       - chain: hypershift-mce-power-test
```
Comment (dharaneeshvrd) [inline ci-operator/step-registry/hypershift/mce/power/conformance/hypershift-mce-power-conformance-workflow.yaml:15]: let's move after test, which ensures the cluster goes to completed state before collecting meaningful info.
```suggestion
      - chain: hypershift-mce-power-test
      - ref: hypershift-mce-agent-info
```

**Review by dharaneeshvrd — CHANGES_REQUESTED** [review]

File: ci-operator/step-registry/hypershift/mce/power/conformance/hypershift-mce-power-conformance-workflow.yaml (hunk 1 above)
Comment (LiangquanLi930) [reply ci-operator/step-registry/hypershift/mce/power/conformance/hypershift-mce-power-conformance-workflow.yaml]: Thanks, Done

**Pull request merged**